package.json

```diff
@@ -43,6 +43,6 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.21.2",
     "jest": "^26.0.1",
-    "prettier": "2.0.5"
+    "prettier": "^2.0.5"
```

Comment:
@atiqueahmedziad you have to rebase this pull request, I merged #6 and this one has now conflicts on package.json.

Reply:
Done :-)